src/components/blog/post/body/share_links.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import PropTypes from "prop-types"
+import classNames from "classnames"
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton,
+} from "react-share"
+
+import useDetectJavascript from "../../../../utils/use_detect_javascript"
+
+import styles from "./share_links.module.scss"
+
+const BlogPostShareLinks = ({ className, url }) => {
+  const hasJavascript = useDetectJavascript()
```

Comment:
Also, I was a bit stupid when I named this. A good alternative should be `hasMounted`, it's the step after the client rehydrates the server-rendered React page, so just keep that in mind